city_scrapers/spiders/chi_ssa_61.py

```diff
@@ -0,0 +1,74 @@
+import re
+from datetime import datetime, time
+
+from city_scrapers_core.constants import COMMISSION
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiSsa61Spider(CityScrapersSpider):
+    name = "chi_ssa_61"
+    agency = "Chicago Special Service Area #60 Hyde Park"
+    timezone = "America/Chicago"
+    allowed_domains = ["www.downtownhydeparkchicago.com"]
+    start_urls = ["http://www.downtownhydeparkchicago.com/about/"]
+    location = {
+        "name": "Polsky Center for Innovation",
+        "address": "1452 East 53rd Street, 2nd Floor, Chicago, IL 60615"
+    }
+
+    def parse(self, response):
+        """Parse meetings on upcoming and minutes pages"""
+        self._validate_location(response)
+        today = datetime.now().replace(hour=0, minute=0)
+        for item in response.xpath('//div[@rel="1"]/ul/li/strong'):
+            text = item.xpath('./text()').extract_first()
+            if not re.search(r"[ pam\.]{2,5}", text):
+                continue
+            start = self._parse_start(text)
+            if not start or (start < today):
+                continue
+            meeting = Meeting(
+                title="commission",
```

Comment:
This should be capitalized as "Commission"

Reply:
fixed this